app/sidekiq/lighthouse/poll_form526_pdf.rb

```diff
@@ -123,10 +123,7 @@ def submission
     end
 
     def get_form526_pdf(submission)
-      user_account = UserAccount.find_by(id: submission.user_account_id) ||
-                     Account.lookup_by_user_uuid(submission.user_uuid)
-
-      icn = user_account.icn
+      icn = submission.account.icn
```

Comment:
Does a missing icn need to be logged?

Reply:
@RachalCassity by virtue of calling `submission.account` the missing icn is logged [here](https://github.com/department-of-veterans-affairs/vets-api/blob/8c9d8fa1b226721eb27152daad08b10e50b0ea55/app/models/form526_submission.rb#L592) if it occurs